src/wallet/userWallet.ts

```diff
@@ -44,11 +52,15 @@ export class UserWallet {
         return new UserWallet({
             kind: UserWalletKind.SecretKey,
             encryptedData,
-            publicKeyWhenKindIsSecretKey: publicKey
+            publicKeyWhenKindIsSecretKey: publicKey,
         });
     }
 
-    static fromMnemonic({ mnemonic, password, randomness }: {
+    static fromMnemonic({
+        mnemonic,
+        password,
+        randomness,
+    }: {
         mnemonic: string;
         password: string;
         randomness?: IRandomness;
```

Comment:
These types could be stored in a separately declared interface. Applicable to the methods above as well. 👍

Reply:
Got it, makes sense. 👍